### PR TITLE
Allow running product tests with arbitrary/downloaded artifacts

### PR DESCRIPTION
**Pull request by ArturGajowy**
by specifying the environment variables:
- PRESTO_SERVER_DIR
- PRESTO_CLI_JAR
- PRODUCT_TESTS_JAR

This is used already in a nightly builds prototype, here:
https://travis-ci.org/ArturGajowy/presto-checks-master/builds/153602756

@fiedukow @kokosing please review :)

### CODE REVIEW

File: presto-product-tests/conf/docker/multinode/compose.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+source ${BASH_SOURCE%/*}/../../../bin/locations.sh
```
Comment (kokosing) [inline presto-product-tests/conf/docker/multinode/compose.sh]: why do you `source` `locations.sh` here?

Comment (ArturGajowy) [inline presto-product-tests/conf/docker/multinode/compose.sh]: it resolves the 
PRESTO_SERVER_DIR
PRESTO_CLI_JAR
PRODUCT_TESTS_JAR
properties (or rahter: their default values). Docker-compose does not have support for default parameter values.

File: presto-product-tests/bin/run.sh [hunk 2]
```diff
@@ -2,24 +2,16 @@
...
 rm -rf "${REPORT_DIR}"
 mkdir -p "${REPORT_DIR}"
 
-source "${PRODUCT_TESTS_ROOT}/target/classes/presto.env"
-
 set +e
 java "-Djava.util.logging.config.file=${PRODUCT_TESTS_ROOT}/conf/tempto/logging.properties" \
     ${PRODUCT_TESTS_JVM_OPTIONS} \
-    -jar "${PRODUCT_TESTS_ROOT}/target/presto-product-tests-${PRESTO_VERSION}-executable.jar" \
+    -jar "${PRODUCT_TESTS_JAR}" \
```
Comment (kokosing) [inline presto-product-tests/bin/run.sh]: reuse run.sh

**Comment by kokosing** [comment]
Add `README.md` section with examples about how to use it.

**Comment by ArturGajowy** [comment]
Updated README, split into more fine-grained commits, rebased on latest master.

**Comment by kokosing** [comment]
looks good

**Comment by ArturGajowy** [comment]
Added a comment to README on PRESTO_SERVER_DIR, PRESTO_CLI_JAR, PRODUCT_TESTS_JAR being optional and defaulting to local build artifacts.

**Comment by ArturGajowy** [comment]
Closing this in favor of FB PR.

**Pull request closed without merging**